manywheel/build_scripts/build.sh

```diff
@@ -44,8 +41,7 @@ autoconf --version
 # against a recent openssl [see env vars above], which is linked
 # statically. We delete openssl afterwards.)
 build_openssl $OPENSSL_ROOT $OPENSSL_HASH
-mkdir -p /opt/python
-build_cpythons $CPYTHON_VERSIONS
+/build_scripts/install_cpython.sh
```

Comment:
Shouldn't the relative path be used here?

Reply:
This script is only used for manywheel so it's fine.